packages/elements/src/item/__test__/item.test.js

```diff
@@ -48,54 +48,68 @@ describe('item/Item', () => {
   describe('Defaults', () => {
     it('Should have correct Shadow DOM structure', async () => {
       const el = await createFixture();
-      expect(el).to.be.accessible();
-      expect(el).shadowDom.to.equalSnapshot();
+      await expect(el).to.be.accessible({
+        ignoredRules: ['aria-allowed-attr']
```

Comment:
To avoid the allowed aria attributes rule of axe-core 4.5 ([aria-not-allow](https://dequeuniversity.com/rules/axe/4.5/aria-allowed-attr?application=axeAPI)). It refers to WAI-ARIA 1.1. So, I checked with WAI-ARAI 1.1 and I found that they still mention about `aria-selected` on some topic but I don't why it violate the rule of the axe-core.